roles/common/tasks/monitoring.yml

```diff
@@ -146,8 +146,16 @@
   sensu_check: name=network_interface_traffic plugin=check-netif.rb state=absent
   notify: restart sensu-client
 
-- name: disk check
-  sensu_check: name=disk plugin=check-disk.rb
+- name: disk check default
```

Comment:
I'm confused by this name. Looking at the arguments I'm guessing you're checking disk levels for everything except `/opt/stack/data`.

But then later you have a specific check for `/`, which is already being checked here, so I'm confused as to the desired outcome.